### PR TITLE
fix(Enjoy2Live): no-cors on recruiting app

### PR DESCRIPTION
**Pull request by Enjoy2Live**

### CODE REVIEW

File: config/middleware.js [hunk 1]
```diff
@@ -5,7 +5,9 @@ const DevLaunchersSubdomains = [
   'https://idea.devlaunchers.com',
   'https://idea-staging.devlaunchers.com',
   'https://projects.devlaunchers.org',
-  'https://projects-staging.devlaunchers.org'
+  'https://projects-staging.devlaunchers.org',
+  'https://dev-recruiters.vercel.app',
+  'https://recruiting-staging.devlaunchers.org',
```
Comment (chungthuang) [inline config/middleware.js:10]: Let's also add `recruiting.devlaunchers.org`.

Comment (Enjoy2Live) [reply config/middleware.js:10]: Good suggestion

Comment (Enjoy2Live) [reply config/middleware.js:10]: Added

**Review by chungthuang — APPROVED** [review]

**Pull request merged**